pepper-apis/src/main/java/org/broadinstitute/ddp/export/ParticipantsResultSetUtil.java

```diff
@@ -0,0 +1,128 @@
+package org.broadinstitute.ddp.export;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang.StringUtils;
+import org.broadinstitute.ddp.content.I18nTemplateConstants;
+import org.broadinstitute.ddp.customexport.export.CustomExporter;
+import org.broadinstitute.ddp.db.dao.ActivityInstanceDao;
+import org.broadinstitute.ddp.db.dao.JdbiMailAddress;
+import org.broadinstitute.ddp.db.dao.ParticipantDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.model.activity.instance.ActivityResponse;
+import org.broadinstitute.ddp.model.address.MailAddress;
+import org.broadinstitute.ddp.model.study.Participant;
+import org.jdbi.v3.core.Handle;
+
+/**
+ * Methods used for extracting {@link Participant}-s data from DB and setting it to List of
+ * {@link Participant}-s.
+ * This data used for exporting (in classes {@link DataExporter}, {@link CustomExporter}.
+ */
+public class ParticipantsResultSetUtil {
```

Comment:
Does anyone else feel like we have way too many "util" classes laying around? I'm curious why we decided to move these out of `ExportUtil`?

Reply:
Moved to ExportUtil. 
My idea was that it is more convenient to have logically connected group of methods in a separate class.